src/traces/sankey/convert-to-d3-sankey.js

```diff
@@ -0,0 +1,80 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Lib = require('../../lib');
+var isArrayOrTypedArray = Lib.isArrayOrTypedArray;
+var isIndex = Lib.isIndex;
+
+module.exports = function(trace) {
```

Comment:
I'd vote for bringing this over to `sankey/calc.js` and remove this file. I don't see the need to have two <100 lines of code file side-by-side.

Reply:
... there's a good reason for it that I can't think of

Reply:
Regardless of where this routine ends up, thanks very much for making sankey a little more _plotly.js-esque_ :tada:

Reply:
Done in 04a186c55523373f7b87eb505bc7c02865ee8be4